m365-developer-proxy/Program.cs

```diff
@@ -17,4 +17,32 @@
 
 rootCommand.Handler = proxyHost.GetCommandHandler(pluginEvents, loaderResults.UrlsToWatch, logger);
 
-return await rootCommand.InvokeAsync(args);
\ No newline at end of file
+// filter args to retrieve short (-n) and long (--name) option aliases
+string[] incomingOptions = args.Where(arg => arg.StartsWith("-") || arg.StartsWith("--")).ToArray();
+
+// compare the incoming options against the root command options
+foreach (Option option in rootCommand.Options)
```

Comment:
We can simplify this with `foreach (var option in ...`

Reply:
Done ✅